helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -70,6 +70,18 @@ rules:
   - secrets
   verbs:
   - "*"
+- apiGroups:
+    - ""
+  resources:
+    - pods
+  verbs:
+    - list
+- apiGroups:
+    - ""
+  resources:
+    - pods/portforward
```

Comment:
Without these permission, operator could not install `chart-operator` inside TC.

Reply:
Ack. That makes sense because app-operator creates a port forward connection using https://github.com/giantswarm/k8sportforward. Once we get rid of tiller we can remove this.